content/responses/folder_locks.yml

```diff
@@ -0,0 +1,18 @@
+---
+title: Folder Locks
+
+type: object
+
+x-box-resource-id: folder_locks
+x-box-tag: folder_locks
+
+description: |-
+  A list of folder locks
+
+allOf:
+  - $ref: "../attributes/pagination.yml"
```

Comment:
Does this actually have pagination? Does this return ever more than 1 lock?

Reply:
It returns some of the pagination elements for future proofing, since multiple locks may be returned. It doesn't return the full object though, so let me adjust this, since only limit and next_marker comes back.

Reply:
Adjusted, that should have been marker_pagination